internal/provider/apppackage.go

```diff
@@ -35,7 +35,7 @@ const (
 	appPackageFileContentKey      = "content"
 )
 
-func resourceAppFile() *schema.Resource {
+func resourceAppPackage() *schema.Resource {
```

Comment:
This was a poorly named function, and should have already been this way. I changed it to make more sense as we now have an identically-named data source and didn't want to cause confusion by mismatched names, or be consistently wrong on both.